main.tf

```diff
@@ -70,14 +71,14 @@ locals {
 # Create Application Service site
 resource "azurerm_template_deployment" "app_service_site" {
   template_body       = "${data.template_file.sitetemplate.rendered}"
-  name                = "${var.product}-${var.env}-webapp"
+  name                = "${var.product}-${var.env}${var.deployment_target}-webapp"
   resource_group_name = "${azurerm_resource_group.rg.name}"
   deployment_mode     = "Incremental"
 
   parameters             = {
-    name                 = "${var.product}-${var.env}"
+    name                 = "${var.product}-${var.env}${var.deployment_target}"
     location             = "${var.location}"
-    env                  = "${var.env}"
+    env                  = "${var.env}${var.deployment_target}"
```

Comment:
Would be better to split this out into parameters which are passed separately into the template e.g. `env` and `deployment_target`. This could lead to potential future bugs where someone expects the `env` parameter to _only_ contain the environment.